### PR TITLE
feat: adopt cephadm

### PR DESCRIPTION
**Pull request by ricolin**
fix https://github.com/vexxhost/atmosphere/issues/711

### CODE REVIEW

File: galaxy.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 namespace: vexxhost
 name: atmosphere
-version: 1.10.3 # x-release-please-version
+version: 1.11.0 # x-release-please-version
```
Comment (mnaser) [inline galaxy.yml:3]: no need to do this, release-please will take care of this

**Comment by ricolin** [comment]
waitting https://github.com/vexxhost/ansible-collection-ceph/actions/runs/7425054349

**Comment by ricolin** [comment]
missed a spot around here https://github.com/vexxhost/ansible-collection-ceph/blob/main/plugins/module_utils/ca_common.py#L80

**Comment by ricolin** [comment]
> missed a spot around here https://github.com/vexxhost/ansible-collection-ceph/blob/main/plugins/module_utils/ca_common.py#L80
fix pr is out: https://github.com/vexxhost/ansible-collection-ceph/pull/24

**Comment by ricolin** [comment]
Idempotence is targeting in https://github.com/vexxhost/ansible-collection-containers/pull/15

**Comment by ricolin** [comment]
just realized I missed rgw that build from rook-ceph currently

**Review by mnaser — CHANGES_REQUESTED** [review]

File: roles/cluster_issuer/tasks/type/self-signed/main.yml [hunk 2]
```diff
@@ -13,6 +13,7 @@
 # under the License.
 
 - name: Create ClusterIssuer
+  changed_when: false
```
[the remark this answers is not in the document]
Comment (ricolin) [reply roles/cluster_issuer/tasks/type/self-signed/main.yml:16]: I think this is introduced by https://github.com/vexxhost/atmosphere/pull/752/files#diff-1ade2795a7b410581d689fb0faa358b769c2d189b2d549e8f530b706f09b2fb7R20
and the only things changed are the update time  in line 173 of attached log file
and resourceVersion in line 178 of attached log file
same thing gose to vexxhost.atmosphere.octavia : Create CAs & Issuers too.

Do you think this is something we should dive into?
[diff.log](https://github.com/vexxhost/atmosphere/files/14407961/diff.log)

**Comment by mnaser** [comment]
@ricolin can you merge or rebase ?

**Pull request merged**